apps/anomalycor/app.css

```
@@ -7,14 +7,29 @@ html, body {
     height: 100%;
 }
 
+@font-face {
+    font-family: 'PublicSans-Regular';
+    src: url("/packages/randyp_mats-common/public/fonts/PublicSans-Regular.ttf");
+}
+
+@font-face {
+    font-family: 'PublicSans-SemiBold';
+    src: url("/packages/randyp_mats-common/public/fonts/PublicSans-SemiBold.ttf");
+}
+
+@font-face {
+    font-family: 'PublicSans-Medium';
+    src: url("/packages/randyp_mats-common/public/fonts/PublicSans-Medium.ttf");
+}
+
 body {
     margin-top: 1%;
     margin-left: 2%;
     margin-right: 2%;
     margin-bottom: 1%;
     padding: 0;
-    background: url(img/bg.png) repeat;
-    font-family: Arial, Helvetica, sans-serif;
+    background: url("/packages/randyp_mats-common/public/img/bg.png") repeat;
+    font-family: PublicSans-Regular, Arial, Helvetica, serif;
```

I think we still want to stay in the sans-serif font family as the final fallback here.

```suggestion
    font-family: PublicSans-Regular, Arial, Helvetica, sans-serif;
```